tests/unit/src/test/scala/tests/CodeLensLspSuite.scala

```diff
@@ -171,6 +171,7 @@ class CodeLensLspSuite extends BaseCodeLensLspSuite("codeLenses") {
            |  def main(args: Array[String]): Unit = ???
            |}""".stripMargin
       )
+      _ <- server.didOpen("a/src/main/scala/Main.scala")
```

Comment:
This is needed so that we save a fingerprint of a working file, which we produce a semanticDB for. SemanticDB has positions that might need to be adjusted in case of a broken/unsaved code and we need the difference between last working and current to figure out how to adjust those positions.